### ISSUE TITLE
[BUG] Running on CentOS 7

### ISSUE THREAD

**Issue by jacobleonwoffenden** [aws-cloudformation/cloudformation-guard#71]
**Describe the bug**
Running `cfn-guard` in a CentOS 7 container produces

```
[root@6ef65b9fa037 /]# cfn-guard 
cfn-guard: /lib64/libc.so.6: version `GLIBC_2.18' not found (required by cfn-guard)
```

**To Reproduce**
Run using 
```
FROM docker.io/centos:7

ENV TOOL_VERSION_CFN_GUARD="1.0.0"

RUN curl -L https://github.com/aws-cloudformation/cloudformation-guard/releases/download/${TOOL_VERSION_CFN_GUARD}/cfn-guard-linux-${TOOL_VERSION_CFN_GUARD}.tar.gz \
      -o cfn-guard-linux-${TOOL_VERSION_CFN_GUARD}.tar.gz \
    && tar -zxf cfn-guard-linux-${TOOL_VERSION_CFN_GUARD}.tar.gz \
    && mv cfn-guard-linux/cfn-guard /usr/local/bin/cfn-guard
```

**Expected behavior**
For `cfn-guard` to run without error

**Screenshots**
n/a

**Operating System:**
CentOS

**OS Version**
7

**Additional context**
Running with the following works fine
```
FROM docker.io/centos:8

ENV TOOL_VERSION_CFN_GUARD="1.0.0"

RUN curl -L https://github.com/aws-cloudformation/cloudformation-guard/releases/download/${TOOL_VERSION_CFN_GUARD}/cfn-guard-linux-${TOOL_VERSION_CFN_GUARD}.tar.gz \
      -o cfn-guard-linux-${TOOL_VERSION_CFN_GUARD}.tar.gz \
    && tar -zxf cfn-guard-linux-${TOOL_VERSION_CFN_GUARD}.tar.gz \
    && mv cfn-guard-linux/cfn-guard /usr/local/bin/cfn-guard
```

**Comment by jacobleonwoffenden** [comment]
`cfn-guard` does run when built with CentOS 7
```
FROM docker.io/centos:7

ENV PATH="${PATH}:/root/.cargo/bin"

RUN yum update -y \
    && yum install -y \
      curl \
      gcc \
      gcc-c++ \
      make \
    && curl https://sh.rustup.rs -sSf | bash -s -- -y
```
```
[root@rust-container workspace]# cargo build --workspace --verbose --release
<omitted cargo build output >

[root@rust-container workspace]# target/release/cfn-guard check --rule_set Examples/ebs-volume-template.ruleset --template Examples/ebs-volume-template.json 
[NewVolume2] failed because [Encrypted] is [false] and the permitted value is [true]
[NewVolume] failed because [Encrypted] is [false] and the permitted value is [true]
[NewVolume] failed because [Size] is [500] and the permitted value is [<= 100]
Number of failures: 3

**Comment by jacobleonwoffenden** [comment]
I'm not quite sure how to proceed with a PR to fix this as I don't know if Travis CI can run builds that aren't Ubuntu, however I have forked, built and released `cfn-guard` using CentOS 7 and GitHub Actions under https://github.com/maglevlabs/cloudformation-guard

**Comment by omkhegde** [comment]
Should be fixed with PR #80

**Comment by jacobleonwoffenden** [comment]
Thanks @omkhegde! 
@jotompki any chance on getting a release created? That should fix #72 as well

**Comment by jotompki** [comment]
Uploaded a centos7 binary to the 1.0.0 release to unblock in the meantime until we get another release out: 

https://github.com/aws-cloudformation/cloudformation-guard/releases/download/1.0.0/cfn-guard-centos-1.0.0.tar.gz
